common/predictive-text/worker/model-compositor.ts

```diff
@@ -103,6 +123,10 @@ class ModelCompositor {
       };
     }
 
+    if(keepOption) {
```

Comment:
I strongly think this is more appropriately done as close to the presentation layer as possible, and _not_ in the `displayAs` string right here.

For example, in KeymanWeb, a suggestion with the `keep` tag can be given the class:

```html
<button class="suggestion keep-suggestion">foo</button>
```

and then style it in CSS:

```css
.suggestion.keep-suggestion::before {
   content: close-quote;
}
.suggestion.keep-suggestion::before {
   content: open-quote;
}
```

And the quotes could be modified per language here:

```
[lang=en] {
  quotes: "“" "”";
}

[lang=fr] {
   quotes: "«" "»";
}
```

Reply:
> I strongly think this is more appropriately done as close to the presentation layer as possible, and _not_ in the `displayAs` string right here.

> And the quotes could be modified per language here:
> 
> ```css
> [lang=en] {
>   quotes: "“" "”";
> }
> 
> [lang=fr] {
>    quotes: "«" "»";
> }
> ```

And that part's the reason it needs to be done in the LMLayer at present - KeymanWeb isn't given access to language modeling data; the quote style is part of that.  Which we'd then need to add as part of a model's specification, and we don't have that designed out quite yet.

I suppose we could add an API to the LMLayer to request the quote types, but then there's dynamically generating the associated CSS.  KMW doesn't really have a problem with doing that - we already dynamically make and link stylesheets at times; I'd just want to make that its own issue/PR, given the extended work required to support it properly.  This one is simple enough at present to be 1 of 3 things addressed here, but such a solution should become its own PR instead.

Would you be fine with making this a separate issue?  The current form could work as a temp stopgap while we pull everything necessary together to do this properly.  Otherwise, I guess I could let this stall and base the adjustment branch off of this PR.

Reply:
Done! Could you please add a `TODO` comment, linking to #1883?